.gitattributes

```diff
@@ -0,0 +1,7 @@
+# See https://git-scm.com/docs/gitattributes for more about git attribute files.
+
```

Comment:
- [ ] Good job by resolving the failed tests as the last reviewer said but there is still one failed test please fix it.
![image](https://user-images.githubusercontent.com/74744787/222928709-3e9e3133-5bf0-4944-8e2b-48ab97e59a49.png)